### PR TITLE
Screen/wishlist

### PR DESCRIPTION
**Pull request by sanjitsarkar**
Added wishlist screen

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -1,11 +1,362 @@
...
+  visibility: hidden;
+}
+nav ul.left,
+nav ul.right {
+  flex-wrap: nowrap;
+}
+header nav ul.left .input-box {
+  min-width: 30vw;
+}
+@media (max-width: 535px) {
```
Comment (manish-GH) [inline css/style.css:27]: use em for media queries

File: css/style.css [hunk 2]
```diff
@@ -1,11 +1,362 @@
...
+  header nav ul.left {
+    width: 100%;
+    display: flex;
+    justify-content: space-between !important;
+  }
+  .search-icon {
+    visibility: visible;
+  }
+}
+@media (max-width: 870px) {
```
Comment (manish-GH) [inline css/style.css:40]: use em

File: css/style.css [hunk 3]
```diff
@@ -1,11 +1,362 @@
...
+  justify-content: center;
+}
+.hero-section {
+  display: flex;
+  width: 100%;
+  padding: 4rem 6rem;
+  align-items: center;
+  justify-content: space-between;
+}
+@media screen and (max-width: 1200px) {
```
Comment (manish-GH) [inline css/style.css:100]: use em

File: css/style.css [hunk 4]
```diff
@@ -1,11 +1,362 @@
...
+  padding: 0.4rem 0.6rem;
+  font-size: 0.75rem;
+  line-height: 1.25rem;
+}
+form.form {
+  background-color: var(--light-grey);
+  margin-bottom: 15vh;
+  margin-top: 0vh;
+}
+@media (max-width: 535px) {
```
Comment (manish-GH) [inline css/style.css:144]: use em

File: css/style.css [hunk 5]
```diff
@@ -1,11 +1,362 @@
...
+  height: 100vh;
+  overflow: hidden;
+  overflow-y: scroll;
+  padding-left: 2%;
+  padding-bottom: 25vh;
+}
+.filter-btn {
+  display: none;
+}
+@media (max-width: 950px) {
```
Comment (manish-GH) [inline css/style.css:199]: use em

File: css/style.css [hunk 6]
```diff
@@ -1,11 +1,362 @@
...
+    width: 100%;
+    left: 0;
+    box-shadow: var(--bx-sh-primary-1);
+  }
+  .filter-btn {
+    display: block;
+  }
+}
+.show {
+  display: flex !important;
```
Comment (manish-GH) [inline css/style.css:229]: why is !important used here?

File: css/style.css [hunk 7]
```diff
@@ -1,11 +1,362 @@
...
+  overflow-y: scroll;
+}
+.apply-btn {
+  padding: 0.75rem 1rem;
+}
+.cart-price-details {
+  background-color: var(--dark-black);
+  width: max-content;
+}
+@media only screen and (max-width: 935px) {
```
Comment (manish-GH) [inline css/style.css:325]: use em

File: css/style.css [hunk 8]
```diff
@@ -1,11 +1,362 @@
...
+  .cart-products {
+    order: 2;
+  }
+
+  .cart-price-details {
+    order: 1;
+  }
+}
+@media only screen and (max-width: 580px) {
```
Comment (manish-GH) [inline css/style.css:334]: use em

File: css/style.css [hunk 9]
```diff
@@ -1,11 +1,362 @@
...
+  .coupon-section input,
+  button {
+    width: 100%;
+  }
+}
+@media only screen and (max-width: 580px) {
```
Comment (manish-GH) [inline css/style.css:340]: use em

**Review by HrishikeshMahalle — APPROVED** [review]
Looks Good to Merge

**Review by manish-GH — COMMENTED** [review]
In the cart screen, you could probably change the color of the + button and swap the + and - buttons.
Avoid using a lot of colors for interaction/CTA. (You have one with red, pink. black, green and blue. Try to limit it to 2-3)

Everything else seems good. Nice work

File: css/style.css [hunk 10]
```diff
@@ -1,11 +1,362 @@
...
+  min-width: 30vw;
+}
+@media (max-width: 535px) {
+  header nav ul.left .input-box {
+    display: none;
+  }
+  header nav ul.left {
+    width: 100%;
+    display: flex;
+    justify-content: space-between !important;
```
Comment (ritikaawasthy) [inline css/style.css:34]: avoid using !important

**Review by ritikaawasthy — APPROVED** [review]
looks good clean code

File: css/style.css (hunk 6 above)
Comment (sanjitsarkar) [reply css/style.css:229]: Else it can't override the default property

**Pull request merged**